packages/components/src/utils/theme-variables.scss

```diff
@@ -23,3 +23,6 @@ $components-color-gray-400: var(--wp-components-color-gray-400, $gray-400);
 $components-color-gray-600: var(--wp-components-color-gray-600, $gray-600);
 $components-color-gray-700: var(--wp-components-color-gray-700, $gray-700);
 $components-color-gray-800: var(--wp-components-color-gray-800, $gray-800);
+
+// Semantic aliases (prefer these over raw gray values when applicable).
+$components-color-border: #{ $components-color-gray-600 };
```

Comment:
I will eventually make this Sass file the source of truth, and make the colors-values.js file read exported values from here.

Reply:
Interested in how is the JS code able to read from SCSS ?

Reply:
> Interested in how is the JS code able to read from SCSS ?

sass-loader can [do this](https://css-tricks.com/getting-javascript-to-talk-to-css-and-sass/#aa-sass-variables-and-javascript) ✨

Reply:
> certainly not all borders. For example it's not to be used for the borders separate the sidebar from the content. Even in the above cases, there are going to be semantic edgecases. I do think at the moment we use it mainly for borders, but I'm still not sure so directly tying it to the term "border" is actually useful.

This is good to know, thanks. You're right, `$components-color-border` is just a convenience variable and gray-600 will continue to exist. I'm still in the process of auditing the codebase, so yes, it could turn out that semantically named variables are not a good idea! Given that base-styles _colors.scss doesn't have any semantic variables, maybe wp-components should lean toward simplicity as well.

> Speaking of, is it worthwhile to update color variables across the codebase, and perhaps retire the _colors.scss file, if theme-variables.scss is going to be the new location?

Not yet. Until we finalize the details of the theming system, we'll be experimenting with our new variables exclusively within the wp-components package. It's likely that base-styles _colors.scss will remain the official source of truth for everything.

Reply:
Thank you for the details!